PartnerSdk/src/main/java/com/microsoft/store/partnercenter/network/PartnerServiceProxy.java

```diff
@@ -393,11 +392,7 @@ public TResponse post( TRequest content )
         RequestBuilder request = RequestBuilder.post( this.buildPartnerServiceApiUri() );
         try
         {
-            request.setEntity( new StringEntity( getJsonConverter().writeValueAsString( content ) ) );
-        }
-        catch ( UnsupportedEncodingException e )
```

Comment:
why are we not expecting any UnsupportedEncodingException to happen here due to the addition of UTF-8 encoding?